docs/source/pages/user_guide/segmentation.rst

```diff
@@ -34,7 +34,7 @@ Following functionalities are provided by :class:`~uds.segmentation.can_segmente
 
         # define Addressing Information for a CAN Node
         can_node_addressing_information = uds.can.CanAddressingInformation(
-            addressing_format=uds.can.CanAddressingFormat.NORMAL_11BIT_ADDRESSING,
+            addressing_format=uds.can.CanAddressingFormat.NORMAL_ADDRESSING,
```

Comment:
Shouldnt be this called "standard" instead of "normal"?

Reply:
actually all addressing names come from ISO-15765-2